oarepo_vocabularies/ui/theme/assets/semantic-ui/less/oarepo_vocabularies_ui/definitions/vocabulary_cf.less

```diff
@@ -0,0 +1,64 @@
+/*******************************
+            Theme
+*******************************/
+
+/*
+  The type is the type of the component as defined in semantic-ui - element, collection, view, module
+  For simple component, 'element' is the correct type
+*/
+@type    : 'modules';
+
+/*
+  This is the name of the component - though not required, keep it the same as the filename
+*/
+@element : 'vocabulary_cf';
+
+
+@import (multiple) '../../theme.config';
+
+/*
+  This is the styling of your component. The class should start with .ui.<component-name>
+
+  Note: use variables here, not the exact values. Placing the values to another file (sample.variables)
+  will allow users to override these values.
+ */
+.ui.vocabulary_cf {
+  h2.ui.header {
+    font-size: 1.2em;
```

Comment:
How to do this better?

Reply:
@mirekys Same here, please have a look. It also includes refactoring of the form and serach app according to the new principle. For detail page search app, this was not possible, as we use direct search kit there, due to the fact that we don't want for search app to influence the URL of the page. Thanks